assets/runtime/functions

```diff
@@ -262,10 +266,15 @@ gitlab_configure_redis() {
   gitlab_finalize_redis_parameters
   gitlab_check_redis_connection
 
+  if [[ -n ${REDIS_PASSWORD} ]]; then
+    REDIS_PASSWORD=':'${REDIS_PASSWORD}'@'
+  fi
+
   update_template ${GITLAB_RESQUE_CONFIG} \
     REDIS_HOST \
     REDIS_PORT \
-    REDIS_DB_NUMBER
+    REDIS_DB_NUMBER \
+    REDIS_PASSWORD
```

Comment:
Here is it the same we should separate it.

Reply:
':'${REDIS_PASSWORD}'@' 
Is the ":" symbol redundant?